src/components/NegotiationList.vue

```diff
@@ -0,0 +1,217 @@
+<template>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
```

Comment:
No need to import font awesome here. Font awesome is already added in main.js. You need to import your icons there.
Have a look at https://fontawesome.com/docs/web/use-with/vue/add-icons